### DISCUSSION TITLE
✅ 🔗 PLEASE can we have global containers? (Duplicate of #24233)

### DISCUSSION THREAD

**Discussion by MontaPresti** [elementor/elementor#25378] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

We need to have global containers, not elements.
If I do an entire block about reviews or a block with several element, why whould I use template across the site and have to search for every instance to modify it at a later time? If I do a block with 10 icon elements and some text and an heading I shouldn't have to create a dozen of single global elements, I need to make the entire container global.

### Describe the solution you'd like.

I'd like to be able to make a container global to scatter it across my website.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by SNCasey1988** [comment]
Hi, did you know that you can save a container as a template and then insert it using shortcode?  Meaning whenever you edit it all the places will still dynamically update with it, like a global element?

**Comment by nicholaszein** [comment]
Hello there @MontaPresti!

This feature already exists with both [Elementor Shortcodes](https://elementor.com/help/how-to-use-shortcodes/#elementor-shortcodes), and the [Template widget](https://elementor.com/help/template-widget-pro/) (Pro).

With these two options you can show any Saved Template from the Template Library on any page. This means, they behave exactly the same as a Header, Footer, or any other Theme Builder templates. You edit them directly from the library, and they update on every page. This is the same behavior of the Global Widgets by the way, just adapted to be used for Library Templates.

Instead of importing a template from the library to your posts and pages, you simply use the Template widget (or a template shortcode), and can update it in one place to be applied to all instances added to your different documents on your website.

You should really take a look, this is a cool feature!

I'm closing this request as solved.

Kind regards
